src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SSPI/ManagedSSPIContextProvider.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NET6_0
```

Comment:
This is probably a bug, or does this class need removal in NET8?

Reply:
It should be removed in NET 8

Reply:
That's why I marked it

Reply:
I think I'm going to remove all these other !NET8_0_OR_GREATER in another PR